README.rst

```diff
@@ -16,6 +15,32 @@ or with financial support from you or your company.
 This is a new project that builds off the exploration done in
 https://github.com/mrocklin/dask-geopandas
 
+Documentation
+-------------
+
+See the documentation on https://dask-geopandas.readthedocs.io/en/latest/
+
+Installation
+------------
+
+This package depends on GeoPandas and Dask. In addition, it is recommended to
+install PyGEOS, to have faster spatial operations and enable multithreading. See
```

Comment:
suggestion:
`In addition, it is recommended to install PyGEOS, to have faster` => `You can also install PyGEOS to have faster...`

Reply:
I would maybe phrase it a bit stronger than "you can also install", as it is really strongly recommended no? (I don't think there is much point in using dask-geopandas without it ..)

Reply:
Well, this sentence is incorrect anyway. We depend on `pygeos` in `calculate_spatial_partitions`.